wiki/Game_modifier/Hidden/id.md

```diff
@@ -0,0 +1,81 @@
+---
+stub: true
+tags:
+  - hidden
+  - HD
+  - mod
+  - game modifier
+---
+
+# Hidden (mod)
+
+![HD mod icon](/wiki/shared/mods/HD.png "Ikon mod Hidden (HD) mod")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Fade In (mod)](/wiki/Game_modifier/Fade_In) ataupun [Flashlight (mod)](/wiki/Game_modifier/Flashlight).*
+
+## Tentang
+
+- Singkatan: HD
+- Tipe: Peningkat Kesulitan
+- Pengali skor:
+  - ![][o!s] ![][o!t] ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- Tombol shortcut default: `F`
+  - Tombol shortcut default ([osu!mania](/wiki/Game_mode/osu!catch)): `F` `F` atau `Shift`+`F`
+- Caption:
+  - ![][o!s]: `Approach circle serta not-not akan memudar/menghilang perlahan-lahan, namun nilai maksimal Anda akan sedikit bertambah.`
+  - ![][o!t]: `Not-not akan menghilang perlahan sebelum anda mengenainya!`
+  - ![][o!c] `Approach circle serta not-not akan memudar/menghilang perlahan-lahan, namun nilai maksimal Anda akan sedikit bertambah.`
+  - ![][o!m]: `Not-not akan menghilang perlahan sebelum anda mengenainya!`
+- Mode-mode permainan yang sesuai: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- Variasi (osu!mania): [Fade In](/wiki/id/Game_modifier/Fade_In)
+
+## Deskripsi
+
+Mod **Hidden** adalah sebuah [game modifier](/wiki/id/Game_modifier) yang meningkatkan kesulitan sebuah beatmap dengan menghilangkan approach circle dan menyebabkan [hit objek](/wiki/id/Hit_object) untuk memudar setelah muncul di layar.
+
+### osu!standard
+
+Pada [osu!standard](/wiki/Game_mode/osu!), mod Hidden melenyapkan approach circle dan menyebabkan hit objek untuk memudar setelah muncul dengan singkat, menyebabkan para pemain untuk kurang-lebihnya mengingat timingnya, dan- pada tingkatan lebih rendah- letak dan jalur slider.
+
+Meskipun sebaiknya diperhatikan bahwa mod Hidden dianggap sebagai mod Peningkat Kesulitan yang paling mudah bagi para top player karena konsistensi pada kapan hit objek-nya muncul dan hilang. Hasil dari konsistensi ini, sangat mungkin untuk mempelajari kapan untuk mengklik hit objek hanya berdasarkan kapan objek tersebut itu menghilang.
+
+![HD gameplay standard](img/HD-standard.jpg "Gameplay osu!standard dengan mod Hidden dinyalakan")
+
+### osu!taiko
+
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), note-note memudar saat mereka sudah di setengah jalan pada layar, menyebabkan para pemain untuk mengingat timing dan warna note nya. Tetapi, slider dan denden tetap berjalan melewati waktu penuh-nya dan tidak memudar, dengan syarat bahwa denden tidak mempunyai approach circle untuk menandakan kapan waktunya habis.
```

Comment:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), not-not memudar saat mereka sudah di setengah jalan pada layar, diharuskan para pemain untuk mengingat timing dan warna note nya. Namun, slider dan denden tetap berjalan melewati waktu penuh dan tidak memudar, dengan syarat bahwa denden tidak mempunyai approach circle untuk menandakan kapan waktunya habis.
```

Reply:
i don't know what is denden lol

Reply:
probably one of the hit object for taiko lol